rails-commenting.rb

```diff
@@ -49,14 +57,17 @@ def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
       # ---8)
+  # This is the route to delete the specific blog post id from the database.
       redirect_to blog_posts_path
     end
   end
 
   # ---9)
+# This is the super secret extra tough Strong Params, for extra security, keeping would be bad actors out of your routes and controllers to protect your database unless specifically allowed. This is meant to prevent Mass Assignment where a bad actor could gain privileges such as admin.  This is basically assigning data to mulitple database sections at the same time which could lead to allowing them access to overwrite data not meant to be changed. 
```

Comment:
This is referring to the word private a not the blog post params. 

You are on the right path for the strong params though!